cmd/vulcan-aws-alerts/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/sirupsen/logrus"
+
+	check "github.com/adevinta/vulcan-check-sdk"
+	"github.com/adevinta/vulcan-check-sdk/state"
+	"github.com/aws/aws-sdk-go/aws/arn"
+)
+
+var (
+	checkName = "vulcan-aws-alerts"
+	logger    = check.NewCheckLog(checkName)
+)
+
+type options struct {
+	VulcanAssumeRoleEndpoint string `json:"vulcan_assume_role_endpoint"`
+	RoleName                 string `json:"role_name"`
+}
+
+func main() {
+	run := func(ctx context.Context, target string, optJSON string, state state.State) error {
+		var opt options
+		if optJSON != "" {
+			if err := json.Unmarshal([]byte(optJSON), &opt); err != nil {
+				return err
+			}
+		}
+		if target == "" {
+			return fmt.Errorf("check target missing")
+		}
+
+		vulcanAssumeRoleEndpoint := opt.VulcanAssumeRoleEndpoint
+		if vulcanAssumeRoleEndpoint == "" {
+			vulcanAssumeRoleEndpoint = os.Getenv("VULCAN_ASSUME_ROLE_ENDPOINT")
+		}
+		if vulcanAssumeRoleEndpoint == "" {
+			return fmt.Errorf("VULCAN_ASSUME_ROLE_ENDPOINT option is missing")
+		}
+
+		roleName := opt.RoleName
+		if roleName == "" {
+			roleName = os.Getenv("ROLE_NAME")
+		}
+
+		parsedARN, err := arn.Parse(target)
+		if err != nil {
+			return err
+		}
+
+		return caCertificateRotation(opt, parsedARN.AccountID, vulcanAssumeRoleEndpoint, roleName, logger, state)
```

Comment:
logger is global! :)

https://github.com/adevinta/vulcan-checks/pull/29/files#diff-f699e45215783525b3a5cdf00299425eR22